modules/ROOT/nav.adoc

```diff
@@ -11,7 +11,6 @@
  ** xref:to-share-api-asset-to-portal.adoc[Share an API Asset to a Public Portal]
  ** xref:to-manage-categories.adoc[Manage Categories]
  ** xref:to-delete-asset.adoc[Delete an Asset]
- ** xref:to-publish-an-asset.adoc[Publish Asset Draft Content]
```

Comment:
Why are we removing the "to publish an asset" link? Is this part of the Nav ReOrg effort?

Reply:
As we added the version agnostic properties info, we saw that all the info in to-publish-an-asset.adoc was covered in to-describe-an-asset.adoc, so we're deleting to-publish-an-asset.adoc in this PR, as a small and easy fix before the navigation reorganization.